video_xblock/video_xblock.py

```diff
@@ -430,6 +486,9 @@ def studio_view(self, context):  # pylint: disable=unused-argument
         self.default_transcripts = filtered_default_transcripts
         if self.default_transcripts:
             self.default_transcripts.sort(key=lambda l: l['label'])
+        # Order basic_fields and advanced_fields
+        basic_fields_ordered = self.order_studio_editor_fields(player.basic_fields)
+        advanced_fields_ordered = self.order_studio_editor_fields(player.advanced_fields)
```

Comment:
Lookin at the code now, I see no need to make basic & advanced fields into dictionaries.

Can you please drop OrderedDict altogether.

And sorry for misguiding you to wrong direction.